libcodechecker/orm_model.py

```diff
@@ -287,6 +266,16 @@ def __init__(self, bug_hash, author, message, created_at):
         self.created_at = created_at
 
 
+class ReviewStatus(Base):
```

Comment:
`bug_hash` perhaps won't be sufficient later on, when `PathHash` is introduced, if it is.,